check_bird_bgp/check_bird_bgp

```diff
@@ -50,7 +51,12 @@ parser.add_argument ('--sessions_down_ok', metavar = "LIST", help = "List of ses
 parser.add_argument ('--sessions_down_ok_file', metavar = "FILENAME", help = "List of sessions which are OK to be down. Provide one interfaces per line.")
 parser.add_argument ('--ignore_missing_file', help = "Ignore a possible non-existent file given as --interfaces_down_ok_file", action = 'store_true')
 parser.add_argument ('--session', help = "Only check for session with given name.")
-
+parser.add_argument ('--imp_w', help = "Warning interval for Imported Routes", metavar = "RANGE")
+parser.add_argument ('--imp_c', help = "Critical interval for Imported Routes", metavar = "RANGE")
+parser.add_argument ('--exp_w', help = "Warning interval for Exported Routes", metavar = "RANGE")
+parser.add_argument ('--exp_c', help = "Critical interval for Exported Routes", metavar = "RANGE")
+parser.add_argument ('--pre_w', help = "Warning interval for Preferred Routes", metavar = "RANGE")
+parser.add_argument ('--pre_c', help = "Critical interval for Preferred Routes", metavar = "RANGE")
```

Comment:
I have renamed these to --routes_imported_w & --routes_exported_c to match the your current args like --ibgp_c

Reply:
Oh sorry, I might have been unprecise here. I added input validation in the master branch and renamed the --i/ebgp_c/w parameters to _warn/_crit as well for readabilty :-) It would be cool, if you could rebase and rename the parameter to the more descriptive version, than I'd be happy to merge the PR. Sorry for any confusion.